cicd/gitops/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc

```diff
@@ -10,8 +10,8 @@ With {gitops-title}, you can configure Argo CD to recursively sync the content o
 
 .Prerequisites
 
-* You have logged in to the `product-title` cluster as an administrator.
-* You have installed the `gitops-title` Operator in your cluster.
+* You have logged in to the {product-title} cluster as an administrator.
+* You have installed the {gitops-title} Operator in your cluster.
```

Comment:
Just a small correction of the mistake I spotted while working on this task. It has nothing to do with the task itself.